girder/web_client/src/views/widgets/ItemListWidget.js

```diff
@@ -140,6 +151,108 @@ var ItemListWidget = View.extend({
         this.checked = _.map(this.$('.g-list-checkbox:checked'), function (checkbox) {
             return $(checkbox).attr('g-item-cid');
         }, this);
+    },
+
+    centerSelected: function () {
+        const widgetcontainer = $('.g-hierarchy-widget-container');
+        const selected = $('li.g-item-list-entry.g-selected');
+
+        if (widgetcontainer.length > 0 && selected.length > 0) {
+            const centerPos = (widgetcontainer.height() / 2.0) + (selected.outerHeight() / 2.0);
+            // Take the current scroll top position and add it to the position of the top of the selected item, then center it
+            const scrollPos = widgetcontainer[0].scrollTop + selected.position().top - centerPos;
+            if (this.tempScrollPos === undefined) {
+                this.tempScrollPos = scrollPos;
+            }
+            // Call a custom scroll event to prevent thinking the user initiated it
+            var e = new CustomEvent('scroll', { detail: 'selected_item_scroll' });
```

Comment:
Not sure what our linter settings are, but it looks like it's not disallowing `var`. Could you change this to `const`?